css-editor/src/Background.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+
+export default class Background extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    }
+  }
+  // setBackgroundColor = (props) => {
+  //   if (this.props.submitted === true) {
+  //     return
+  //     <div classname='background' style={{ backgroundColor: `${props.backgroundColor}` }}>
+  //     </div>
+  //   } else {
+      
+  //   }
+  // }
```

Comment:
Commented out code should be removed from production but once again since were in a hackathon not sure if the convention applys here.